### DISCUSSION TITLE
Add ability to multiple signal by constant in plot panel

### DISCUSSION THREAD

**Discussion by bertaveira** [foxglove/community#729] [Q&A]
I like that there are math operations supported by the plot panel but what I find most useful is not supported. Multiplying the signal by a constant.

Example is converting from radians to degrees for better visualisation, flipping sign by multiplying by -1, in a robot with a gear ratio multiply by the ratio to match the other sensor on the other side of the gearing... etc

**Comment by jtbandes** [comment]
See also: https://github.com/orgs/foxglove/discussions/425
